src/components/admin/FileButton.tsx

```diff
@@ -0,0 +1,18 @@
+import Button from '@mui/material/Button'
+import theme from "../../theme";
+
+interface Props {
+    handleFileInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+export const FileButton = ({ handleFileInputChange }: Props) => (
+    <Button component="label" sx={{ color: theme.palette.primary.main }}>
+        Dodaj plik
+        <input
+            type="file"
+            hidden
```

Comment:
tu tylko taka uwaga czysto dekoracyjna. W pracy w wszystkie atrybuty które są boolen i nie mają nic do siebie przypisane, ustawiam na dole. A np akcje na zmiane jak onChange zawsze na górze, czyli wtedy wygląda to tak
`<input`
            `onChange={handleFileInputChange}`
           `type="file"`
            `accept=".csv"`
            `hidden`
       ` />`
      Ja sam nie przestrzegałem tego jeszcze w tym projekcie i teraz też nie trzeba tego zmieniać, ale to jest cos na co możemy zwrócić uwagę na planowaniu, czy gdzieś zapisać jako zasada pisania atrybutów w komponentach. Wtedy cała aplikacja bedzie przejrzysta i spójna  😊

Reply:
dzięki za podpowiedź, zastosowana :)